src/global/dom/input-element.ts

```diff
@@ -1,16 +1,23 @@
+import { findReferencedElement } from './find-referenced-element';
+
 /**
  * Resolves the input element inside the shadow DOM of the given element.
  */
-export function inputElement(element: HTMLElement): HTMLInputElement | null {
+export function shadowInputElement(element: HTMLElement): HTMLInputElement | null {
```

Comment:
Should this be named `findShadowInput` to be consistent with the other function?